### DISCUSSION TITLE
Method to create printer port

### DISCUSSION THREAD

**Discussion by ghost** [mhammond/pywin32#2517] [General]
Is there any method to create a standard TCP/IP printer port? If yes, can you please guide me in that direction. If not, it would be nice to have it to have more automation in printer area.

Thanks.

Reported by: gaurav517

Original Ticket: [pywin32/feature-requests/115](https://sourceforge.net/p/pywin32/feature-requests/115)

**Comment by Avasam** [comment]
Closing as original poster does not exist
